pkg/reconciler/instances/ory/test/ory_integration_test.go

```diff
@@ -78,3 +31,74 @@ func TestOryIntegration(t *testing.T) {
 		require.Equal(t, true, ready)
 	}
 }
+
+func TestOryIntegrationProduction(t *testing.T) {
+	skipTestIfDisabled(t)
+
+	if !isProductionProfile() {
+		t.Skipf("Integration tests disabled: skipping parts of test case '%s'", t.Name())
+	}
+
+	setup := newOryTest(t)
+	defer setup.contextCancel()
+
+	t.Run("ory-hydra on production profile is deployed", func(t *testing.T) {
```

Comment:
How about putting duplicated phrase: `on production profile is deployed` above all test cases? Basically that's the aim of the test. Additional `t.Run("Ensure that on production profile (...)")` or something similar may be cool!